src/index.js

```diff
@@ -126,6 +132,12 @@ const CLOUD_RESOURCE_HEADER = 'google-cloud-resource-prefix';
  */
 const MAX_REQUEST_RETRIES = 5;
 
+/*!
+ * The maximum number of concurrent requests supported by a GAPIC client.
```

Comment:
Can you clarify that this is a limitation of the GAPIC client (as opposed to a maximum we've decided to impose)?  And if there's any docs / code you can link to regarding the limit, that would be useful context too.

Reply:
I have only seen this confirmed in a Google groups post and this is actually a limit in both the GFE and the clients. I expanded on the comment and further shifted the blame on everyone else but us.